cmd/main.go

```diff
@@ -129,7 +129,11 @@ func main() {
 		NSXConfig: cf,
 	}
 
-	// Start the security policy controller.
+	if cf.CoeConfig.EnableVPCNetwork && commonService.NSXClient.NSXCheckVersionFor(nsx.FeatureVPC) {
+		log.V(1).Info("VPC mode enabled")
+		// Start controllers which only supports VPC
+	}
+	// Start the security policy controller, it supports VPC and non VPC mode
```

Comment:
if cf.CoeConfig.EnableVPCNetwork is false, we also need to start security policy controller because securitypolicy controller can also work in non vpc mode.

Reply:
Yes, it is as expected now.

Reply:
I see. I miss code line #L138.